### PR TITLE
Fixed calibration file doesn't exist error

### PR DESCRIPTION
**Pull request by jaladh-singhal**
While calculating vega magnitudes, calibration file present as `wsynphot/data/calibration/alpha_lyr_mod_002.fits` is shown to be missing from the built package. The reason is, it is not being included in package_data in `setup.py` - since `data/*` doesn't work recursively (unless there's a init.py in directory) so directories like `calibration` are missing in built package. So to extend depth to one level more, I added `data/*/*` and it fixed the problem and this is the maximum depth of files in `wsynphot/data` so we need not to go further.

Also the IOError message shows to download the missing calibration file using function `download_calibration()` which was removed under https://github.com/starkit/wsynphot/commit/3bb2c9c8ad7ef2b7b040f7577ffaa46e0541ac6f#diff-e39e153dbbd3a0ecb13c9a072d53a0a0 and function`download_all()` was never defined. So it's misleading message for users when they encounter such an error. Also I found a commit https://github.com/starkit/wsynphot/commit/49d464aacd802d68b88fbea7b3322e7cfcb269d1 (which was committed after this function removal commit) whose message says `modification of instructions ` but there's no such change - so it appears to me that error message correction was left by mistake.

### CODE REVIEW

File: wsynphot/calibration.py [hunk 1]
```diff
@@ -14,9 +14,7 @@ def get_vega(vega_file=None):
     if vega_file is None:
         vega_file = default_vega_path
     if not os.path.exists(vega_file):
-        raise IOError('Calibration file {0} does not exist - please download by'
```
Comment (wkerzendorf) [inline wsynphot/calibration.py]: why not doing the download?

Comment (jaladh-singhal) [reply wsynphot/calibration.py]: @wkerzendorf I've added the download_calibration() function by using same code of commit I shared https://github.com/starkit/wsynphot/commit/3bb2c9c8ad7ef2b7b040f7577ffaa46e0541ac6f#diff-e39e153dbbd3a0ecb13c9a072d53a0a0 in PR description, with some required changes! Let me know if this is what you wanted?

**Comment by jaladh-singhal** [comment]
The build fails not because of this function, but because get_filter_data parameterized tests are failing because of a recent SVO update, which I've fixed in https://github.com/starkit/wsynphot/pull/34. Let's merge that PR 1st & re-run the build for this PR.

File: wsynphot/data/base.py [hunk 2]
```diff
@@ -16,6 +16,8 @@
 ALPHA_LYR_PATH = os.path.join(os.path.dirname(__file__), 'calibration',
```
Comment (wkerzendorf) [inline wsynphot/data/base.py]: this should live in the data directory with the filter sets.

Comment (jaladh-singhal) [reply wsynphot/data/base.py]: Changed that

File: wsynphot/calibration.py [hunk 3]
```diff
@@ -3,23 +3,30 @@
...
 from wsynphot.spectrum1d import SKSpectrum1D as Spectrum1D
-from wsynphot.data.base import ALPHA_LYR_FNAME, ALPHA_LYR_PATH
+from wsynphot.data.base import ALPHA_LYR_PATH
 
 default_vega_path = ALPHA_LYR_PATH
 
-def get_calibration_dir(fname):
-    return os.path.join(wsynphot.__path__[0], 'data', 'calibration', fname)
-
 def get_vega(vega_file=None):
```
Comment (wkerzendorf) [inline wsynphot/calibration.py]: call it get_vega_calibration_spectrum

**Pull request merged**